app/uk/gov/hmrc/transitmovements/repositories/DeparturesRepository.scala

```diff
@@ -62,15 +63,28 @@ import scala.util.control.NonFatal
 @ImplementedBy(classOf[DeparturesRepositoryImpl])
 trait DeparturesRepository {
```

Comment:
This should become `MessagesRepository`

Reply:
Yes, I was planning to rename this to MovementsRepository.  And to the other point @drnaylor-oc  mentioned, yes my next PR will include these changes to rename Departure to Movement and delete DepartureId and use MovementId instead.

Reply:
Yes, MovementsRepository, not MessagesRepository, silly me! I'm always getting those two confused when writing them down.

With regards to movement type -- this PR is just in a slightly weird state where we have Departure being supplied with a MovementType which is odd and makes me hesitant to merge as it stands because it it doesn't make sense. Similarly, you've moved the collection over to be "movements" and changed the model, but you haven't changed everything else around it.

I'd suggest making the rename changes now to make that consistent with the collection and model changes.

Reply:
yep, thought of doing this incrementally first, it's  better to rename theses changes as part of this PR  to be consistent.